requirements-dev.txt

```diff
@@ -1,4 +1,3 @@
-# coveragepy==5.0 fails with `Safety level may not be changed inside a transaction
```

Comment:
I didn't think this was necessary to have, since I don't see a situation where we would downgrade `coveragepy` back to v5